tests/unit/peapods/pods/test_compound_pods.py

```diff
@@ -118,21 +118,33 @@ def test_pod_naming_with_parallel(runtime):
 
         assert bp.replicas[0].name == 'pod/0'
         assert bp.replicas[0].peas[0].name == 'pod/0/head'
+        assert bp.replicas[0].peas[0].inner is False
```

Comment:
this should go in another test or the test needs to be renamed

Reply:
I don't think that's really useful. Because you need to know which ones to assert are `inner` or not, so you need to keep track of the names anyway. I'd keep them together